core/src/solution_submission.rs

```diff
@@ -219,8 +225,9 @@ async fn retry_with_gas_price_increase(
             )
             .await;
 
+        let gas_price_can_still_increase = gas_price < effective_gas_cap;
```

Comment:
very cleaver!

Reply:
Clever, but what does it bring to the table apart from readability? I see it is only used in one place...

Reply:
I like this because it gives the condition a name. It makes it clear what `gas_price < effective_gas_cap` means on a higher level than just what it does.